lib/logging_bugfender.dart

```diff
@@ -96,8 +100,8 @@ class LoggingBugfenderListener {
       return FlutterBugfender.setDeviceBool(key, value);
     } else if (value is double) {
       return FlutterBugfender.setDeviceFloat(key, value);
-    } else if (value is int) {
-      return FlutterBugfender.setDeviceInt(key, value);
+    } else if (value is num) {
```

Comment:
because of [this lint](https://dart-lang.github.io/linter/lints/avoid_double_and_int_checks.html)